### PR TITLE
Expose (if enabled) prometheus metrics over http (#815).

### PR DESCRIPTION
**Pull request by kuba--**
Signed-off-by: kuba-- <kuba@sourced.tech>

Closes https://github.com/src-d/gitbase/issues/815
 
 - [X] I updated the documentation explaining the new behavior if any.
 - [X] I updated CHANGELOG.md file adding the new feature or bug fix.
 - [X] I updated go-mysql-server using `make upgrade` command if applicable.
 - [X] I added or updated examples if applicable.
 - [X] I checked that changes on schema are reflected into the documentation, if applicable.

### CODE REVIEW

File: internal/function/uast.go [hunk 1]
```diff
@@ -263,6 +289,8 @@ func (u *uastFunc) getUAST(
 		return nil, nil
 	}
 
+	finish(ok)
```
Comment (erizocosmico) [inline internal/function/uast.go]: what happens if there is an early return?

Comment (kuba--) [reply internal/function/uast.go]: early return happens on error, and here we build uast histogram (measure latency), so if we collect immediately returns on errors then final histogram  may be incorrect. That's why we measure only successfully queries, here.
If we want, we can count errors, but I think it should be done on uast side, but it's up to us.

**Comment by ajnavarro** [comment]
LGTM after rebase @kuba--

**Comment by ajnavarro** [comment]
@kuba-- could you rebase please?

**Comment by kuba--** [comment]
PTAL a new PR: https://github.com/src-d/gitbase/pull/919

**Pull request closed without merging**